src/applications/vaos/components/tests/PhoneLayout.unit.spec.js

```diff
@@ -0,0 +1,375 @@
+          name: /Phone appointment/i,
+        }),
+      );
+      expect(screen.getByRole('heading', { level: 2, name: /How to join/i }));
+      expect(screen.getByRole('heading', { level: 2, name: /When/i }));
+      expect(
+        screen.container.querySelector('va-button[text="Add to calendar"]'),
+      ).to.be.ok;
+
+      expect(screen.getByRole('heading', { level: 2, name: /What/i }));
+      expect(screen.getByText(/Primary care/i));
+
+      expect(
+        screen.getByRole('heading', {
+          level: 2,
+          name: /Scheduling facility/i,
+        }),
+      );
+      expect(screen.getByText(/Cheyenne VA Medical Center/i));
+      expect(screen.getByText(/2360 East Pershing Boulevard/i));
+      expect(screen.container.querySelector('va-icon[icon="directions"]')).not
+        .to.exist;
+
+      expect(screen.getByText(/Clinic:/i));
+      expect(screen.getByText(/Clinic 1/i));
+      expect(screen.getByText(/Clinic phone:/i));
+      expect(
+        screen.container.querySelector('va-telephone[contact="307-778-7550"]'),
+      ).to.be.ok;
+
+      expect(
+        screen.getByRole('heading', {
+          level: 2,
+          name: /Details you shared with your provider/i,
+        }),
+      );
+      expect(screen.getByText(/Reason:/i));
+      expect(screen.getByText(/This is a test/i));
+      expect(screen.getByText(/Other details:/i));
+      expect(screen.getByText(/Additional information/i));
+
+      expect(screen.container.querySelector('va-button[text="Print"]'));
+      expect(
+        screen.container.querySelector('va-button[text="Cancel appointment"]'),
+      ).to.be.ok;
+    });
+
+    it('should display default text for empty data', async () => {
+      // Arrange
+      const store = createTestStore(initialState);
+      const appointment = {
+        location: {},
+        videoData: {},
+        vaos: {
+          isCommunityCare: false,
+          isCompAndPenAppointment: false,
+          isCOVIDVaccine: false,
+          isPendingAppointment: false,
+          isUpcomingAppointment: true,
+          isVAPhoneAppointment: true,
+          apiData: {},
+        },
+        status: 'booked',
+      };
+
+      // Act
+      const screen = renderWithStoreAndRouter(
+        <PhoneLayout data={appointment} />,
+        {
+          store,
+        },
+      );
+
+      // Assert
+      expect(screen.getByRole('heading', { level: 2, name: /What/i }));
+      expect(screen.getByText(/Type of care information not available/i));
+
+      expect(
+        screen.getByRole('heading', { level: 2, name: /Scheduling facility/i }),
+      );
+      expect(screen.getByText(/Facility details not available/i));
+      expect(screen.container.querySelector('va-icon[icon="directions"]')).not
+        .to.exist;
+      expect(
+        screen.getByRole('link', {
+          name: /Find facility information Link opens in a new tab/i,
+        }),
+      );
+
+      // TODO: Check with Ciera on this.
```

Comment:
Are you planning to resolve this before merging this PR?

Reply:
@JunTaoLuo - Done